### PR TITLE
Re-export fuel_tx in fuels

### LINKED ISSUE #278: Re-export `fuel-tx` in prelude
The SDK only works with specific versions of `fuel-tx`, so it should be re-exported in the prelude.

### PR DESCRIPTION
**Pull request by Br1ght0ne**
Close #278. Might use some items in `prelude`.

### CODE REVIEW

File: packages/fuels/src/lib.rs [hunk 1]
```diff
@@ -32,6 +32,10 @@ pub mod test_helpers {
     pub use fuels_test_helpers::*;
 }
 
+pub mod tx {
+    pub use fuels_tx::*;
```
Comment (Voxelot) [inline packages/fuels/src/lib.rs:36]:
```suggestion
    #[doc(no_inline)]
    pub use fuel_tx::*;
```

**Comment by Voxelot** [comment]
Maybe we could re-export fuel-tx from fuels-core into the prelude instead of creating a new dep here? Otherwise we'll have to bump the version in several places every upgrade.

**Comment by Br1ght0ne** [comment]
> Maybe we could re-export fuel-tx from fuels-core into the prelude instead of adding a new dep in fuels? Otherwise we'll have to bump the version in several places every upgrade.

Good call! I'm less oriented in the dep tree :)

**Comment by vnepveu** [comment]
I agree with Brandon here

**Comment by adlerjohn** [comment]
Poking @digorithm and @vnepveu for review

**Review by Voxelot — APPROVED** [review]

**Review by adlerjohn — APPROVED** [review]

**Review by digorithm — APPROVED** [review]

**Pull request merged**